File: 수학/2609.py
import math
N, M = map(int, input().split())

# 내장함수 math.gcd : 최대공약수(Greatest Common Divisor)
print(math.gcd(N, M))
# 내장함수 math.lcm : 최소공배수(Least Common Multiple)
print(math.lcm(N, M))

File: 수학/2609.cpp
#include <iostream>
using namespace std;

int gcd(int a, int b){
    int r = a%b;
    while(r != 0){
        a = b;
        b = r;
        r = a%b;
    }
    return b;
}

int main(){
    int N, M;
    cin >> N >> M;
    cout << gcd(N, M) << endl;
    cout << (N*M) / gcd(N, M) << endl;
}